src/Parsers/GeoCoordinateParser.php

```diff
@@ -113,7 +112,7 @@ public static function areCoordinates( $string ) {
 		static $parser = null;
 
 		if ( $parser === null ) {
-			$parser = new self( new ParserOptions() );
```

Comment:
Does this work with DataValues 0.1 as well? If not, the dependency should be updated first